server/accounts.go

```diff
@@ -53,12 +53,14 @@ type Account struct {
 	lqws         map[string]int32
 	usersRevoked map[string]int64
 	actsRevoked  map[string]int64
+	respMap      map[string][]*serviceRespEntry
```

Comment:
Just thinking of config reload where we do transfer things between account objects. Is that relevant here? (not necessarily the content of this map but the fact that we should move this map to the new account. I am thinking this kind of thing: https://github.com/nats-io/nats-server/blob/master/server/reload.go#L886).

Reply:
These are runtime only, but if we reload we should transfer them over if we do not keep the live ones but make new ones.

Reply:
ok so I think this may be an issue. It looks like we blow away all the accounts and reload them from the configuration. Note this is only for configured accounts, this does not apply to any resolver based accounts, even those from the configuration.

Reply:
Will fix.

Reply:
ok take another look.